_data/projects/30SecondsOfC++.yml

```diff
@@ -0,0 +1,11 @@
+name: 30 Seconds of C++
+desc: A collection of STL features (functions/libraries) of C++ which can be learned in 30 seconds or less.
+site: https://github.com/Bhupesh-V/30-seconds-of-cpp
+tags:
+- c++
+- c++11
+- c++14
+- c++17
+upforgrabs:
+  name: good first contribution
+  link: https://github.com/Bhupesh-V/30-seconds-of-cpp/issues
```

Comment:
@Bhupesh-V You don't see to have a label by the name of "good first contribution", please create a new label for this name or use labels from herehttps://github.com/Bhupesh-V/30-seconds-of-cpp/labels. Also, please add the label link for eg: https://github.com/Bhupesh-V/30-seconds-of-cpp/labels/good%20first%20issue

Reply:
Ok 
Can I add multiple issue labels or it's just limited to only one ??

Reply:
Only one

Reply:
@Bhupesh-V

Reply:
Does it need ay other changes ?? @ritwik12

Reply:
Yep, change the link please, https://github.com/Bhupesh-V/30-seconds-of-cpp/labels/beginner-friendly